src/core/common/Buttons/CustomShowButton.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { ShowButton } from "react-admin";
+
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+    showButton: {
+        color: '#30ad57',
+        fontSize: 16,
+        textTransform: 'capitalize',
```

Comment:
Default value of React-admin ShowButton is uppercase (it will be if we remove _textTransform: 'capitalize'_ from **styles**): 
![image](https://user-images.githubusercontent.com/39263134/54189669-f239f080-44ba-11e9-8d90-e05bed611528.png)

I need in lower case with the first capital letter, for this reason I used "capitalize":
![image](https://user-images.githubusercontent.com/39263134/54189792-34633200-44bb-11e9-83b7-d601b769641d.png)

Reply:
Right, thanks for the clarification.